cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/notifier/AlertaSelfHealingNotifier.java

```diff
@@ -102,112 +117,40 @@ public void alert(Anomaly anomaly, boolean autoFixTriggered, long selfHealingSta
-                alertaMessage.setCreateTime(outFormatter.format(Instant.ofEpochMilli(topicAnomaly.detectionTimeMs())));
-                alertaMessages.add(alertaMessage);
-              }
-            } else if (topicAnomaly instanceof TopicReplicationFactorAnomaly) {
-              TopicReplicationFactorAnomaly topicReplicationFactorAnomaly = (TopicReplicationFactorAnomaly) topicAnomaly;
-              for (Entry<Short, Set<TopicReplicationFactorAnomalyEntry>> entry : 
-                topicReplicationFactorAnomaly.getBadTopicsByReplicationFactor().entrySet()) {
-                entry.getValue().forEach(topicReplicationFactorAnomalyEntry -> {
-                  AlertaMessage alertaMessage = new AlertaMessage(localHostname,
-                      "TOPIC_REPLICATION_FACTOR_ANOMALY - " + topicReplicationFactorAnomalyEntry.topicName());
-                  alertaMessage.setSeverity(severityFromPriority(anomalyType.priority()));
-                  alertaMessage.setGroup("Performance");
-                  alertaMessage.setValue(String.format("%.2f", topicReplicationFactorAnomalyEntry.violationRatio()));
-                  alertaMessage.setCreateTime(outFormatter.format(Instant.ofEpochMilli(topicAnomaly.detectionTimeMs())));
-                  alertaMessages.add(alertaMessage);
-                });
-              }
-            } else {
-              AlertaMessage alertaMessage = new AlertaMessage(localHostname,
-                  anomalyType.toString());
-              alertaMessage.setSeverity(severityFromPriority(anomalyType.priority()));
-              alertaMessage.setGroup("Performance");
-              alertaMessage.setCreateTime(outFormatter.format(Instant.ofEpochMilli(topicAnomaly.detectionTimeMs())));
-              alertaMessages.add(alertaMessage);
-            }
+            alertTopicAnomaly(anomalyType, localHostname, outFormatter, alertaMessages, topicAnomaly);
             break;
           case MAINTENANCE_EVENT:
             MaintenanceEvent maintenanceEvent = (MaintenanceEvent) anomaly;
-            AlertaMessage alertaMessage = new AlertaMessage(localHostname, anomalyType.toString() + " - " + maintenanceEvent.reasonSupplier());
-            alertaMessage.setSeverity(severityFromPriority(anomalyType.priority()));
-            alertaMessage.setGroup("Performance");
-            alertaMessage.setCreateTime(outFormatter.format(Instant.ofEpochMilli(maintenanceEvent.detectionTimeMs())));
-            alertaMessages.add(alertaMessage);
+            alertMaintenanceEvent(anomalyType, localHostname, outFormatter, alertaMessages, maintenanceEvent);
             break;
           default:
             throw new IllegalStateException("Unrecognized anomaly type.");
         }
 
         for (AlertaMessage alertaMessage : alertaMessages) {
           alertaMessage.setEnvironment(_alertaEnvironment);
-          alertaMessage.setService(Arrays.asList("cruise-control"));
+          alertaMessage.setService(Arrays.asList(ALERT_CRUISE_CONTROL));
```

Comment:
Nit: Since it is a singleton, can we use `Collections.singletonList` rather than `Arrays.asList`?